crates/rpc/rpc/src/eth/filter.rs

```diff
@@ -573,6 +755,8 @@ struct ActiveFilter<T> {
     last_poll_timestamp: Instant,
     /// What kind of filter it is.
     kind: FilterKind<T>,
+    /// Reorg blocks that are relevant to this filter
+    reorg_blocks: Arc<RwLock<HashMap<BlockHash, Vec<Log>>>>,
```

Comment:
this should be a regular parking_lot::Rwlock because only holds memory and access is short lived